script/test-template.sh

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env bash
-set -e  # Exit immediately if a command returns a non-zero status
+set -Ee # Exit immediately if a command returns a non-zero status
```

Comment:
What does the capital `E` do in here? 

From my understanding. `-E` will run a function before the shell exits if an error occurs. But we aren't running anything on Error, just a normal shell exit.

Reply:
It's not strictly required but I think is good practice to have Bash setup as strictly as possible. The above I did almost mindlessly because I'm using `-Eeuo pipefail` for all my Bash since a while, in this case adding `-E`  doesn't hurt and it might prevent unexpected behavior down the line. I can remove if you think is not a good idea.

Reply:
Sounds good to keep it.